frontend/src/static/js/v_summary.js

```diff
@@ -388,6 +388,7 @@ window.vSummary = {
       this.filtered = full;
 
       this.sortFiltered();
+      deactivateAllOverlays();
```

Comment:
Why we put this line here but not in `getFilterd()`?

Reply:
This is inside `getFiltered()`:
https://github.com/reposense/RepoSense/blob/bf2fa03a34da18fc89112efc1c2ef544b716c699/frontend/src/static/js/v_summary.js#L356-L392

Reply:
OK, got it.